api/clients/v2/verification/cert_verifier.go

```diff
@@ -184,26 +184,6 @@ func (cv *CertVerifier) GetQuorumNumbersRequired(ctx context.Context) ([]uint8,
 	return quorumNumbersRequired, nil
 }
 
-// GetRelayRegistryAddress returns the address of the EigenDARelayRegistry contract
-func (cv *CertVerifier) GetRelayRegistryAddress(ctx context.Context) (*gethcommon.Address, error) {
```

Comment:
this will require a change in the proxy when doing dependency bump: http://github.com/Layr-Labs/eigenda-proxy/blob/100801d1dcb4cb3d1d36455da3fad5ef6f3fe34f/store/storage_manager_builder.go#L226-L229

Reply:
It's just a compilation issue though, right? Not actually being used for anything?

Reply:
Actually, thinking about bindings some more, shouldn't we be manually generating MINIMAL interfaces?
I just started wondering why we generated ABIs for the entire contract when individual call sites typically only need to make 1 or 2 calls, and hence only need a small subset of the ABI.
This is basically the golang interface philosophy. Define the interface at each call site. Also protects from breaking changes to the contract that I don't care about.

There's def some tooling missing to make this approach secure, but I feel like even the full-ABI generation is not super secure the way its used right now (see https://x.com/samlafer/status/1917127140090515556)

Reply:
Right! I touch on that topic [here](https://github.com/Layr-Labs/eigenda/pull/1492#discussion_r2066151413)